site-src/api-types/backendtlspolicy.md

```diff
@@ -0,0 +1,133 @@
+# BackendTLSPolicy
+
+??? example "Experimental Channel in v1.0.0+"
+
+    The `BackendTLSPolicy` resource is Alpha and part of the Experimental Channel in `v1.0.0+`.
+
+!!! info "Experimental Channel"
+
+    The `BackendTLSPolicy` resource described below is currently only included in the "Experimental" channel
+    of Gateway API. For more information on release channels, refer to the [related documentation](https://gateway-api.sigs.k8s.io/concepts/versioning).
+
+
+[BackendTLSPolicy][backendtlspolicy] is a Gateway API type for specifying the TLS configuration
+of the connection from the Gateway to a backend pod/s via the Service API object.
+
+## Background
+
+`BackendTLSPolicy` specifically addresses the configuration of TLS in order to convey HTTPS from the Gateway
+dataplane to the backend.  This is referred to as "backend TLS termination" and enables the Gateway to know
+how to connect to a backend pod that has its own certificate.  
+
+While there are other API objects provided for TLS to be configured for **passthrough** and **edge** termination,
+this API object allows users to specifically configure **backend** TLS termination.  For more information on TLS
+configuration in Gateway API, see [TLS Configuration](/guides/tls/).
+
+![Image showing three Termination Types](/images/tls-termination-types.png)
+
+BackendTLSPolicy is a Direct [PolicyAttachment](/reference/policy-attachment/) without defaults or overrides,
+applied to a Service that accesses a backend, where the BackendTLSPolicy resides in the same namespace as the
+Service to which it is applied. The BackendTLSPolicy and the Service must reside in the same namespace in order
+to prevent the complications involved with sharing trust across namespace boundaries.  
+
+One can reuse the same BackendTLSPolicy for all the different Routes that might point to a referenced Service.
+
+## Spec
+
+The specification of a [BackendTLSPolicy][backendtlspolicy] consists of:
+
+- [TargetRef][targetRef] - Defines the targeted API object of the policy.  Only Service is allowed.
+- [TLS][tls] - Defines the configuration for TLS, including hostname, CACertRefs, and WellKnownCACerts.
+- [Hostname][hostname] - Defines the Server Name Indication (SNI) that the Gateway uses to connect to the backend.
+- [CACertRefs][caCertRefs] - Defines one or more references to objects that contain PEM-encoded TLS certificates,
+which are used to establish a TLS handshake between the Gateway and backend Pod.  Either CACertRefs or WellKnownCACerts
+may be specified, but not both.
+- [WellKnownCACerts][wellKnownCACerts] - Specifies whether system CA certificates may be used in the TLS
+handshake between the Gateway and backend Pod.  Either CACertRefs or WellKnownCACerts may be specified, but not both.
+
+![Image showing BackendTLSPolicy objects](images/backendtlspolicy-api.png)
```

Comment:
This diagram is a bit confusing to me. Maybe it's just that it's hard to distinguish the field name with the field type. I'd consider omitting it for now and adding it in a follow up. For any diagrams that aren't just mermaid.js, is it possible to share a link to a source file that others can copy to modify?

Reply:
I'm going to share some modified mermaid for this diagram.  I can still omit it if you think it doesn't add any value.  There is one other diagram here (TLS Termination Types) that exists already in the repo.  I can try to find the original source.

Reply:
@robscott I also just updated the mkdocs.yaml and alphabetized the API types there.  I didn't want BackendTLSPolicy to be the first in the list, so I added a suboption called Policy, where we can put more Policies in the future.